routes/blog.js

```diff
@@ -0,0 +1,72 @@
+const routes = require('express').Router();
+const blogModel = require('../models').Blog;
+ 
+// Get All Blogs
+routes.get('/', async (req, res) => {
+    try{
+        const blogs = await blogModel.findAll();
+        res.status(200).json({'data': blogs });      
+    }catch(err){
+        res.status(500).json({'error': err });
+    }
+});
+
+// Create Blog
+routes.post('/create', async (req, res) => {
+    
+    try{
+        const newBlog = await blogModel.create({ 
+            title: req.body.title,
+            description: req.body.description,
+            userId: req.body.userId,
+            status: req.body.status
+        });
+        res.json({'data': newBlog});
+    }catch(err){ 
+        res.json({'data': err});
+    }
+});
+
+// Get One Blog by its Id
+routes.get('/:id', async (req, res) => {
+    let blog = await blogModel.findOne({ where: { id: req.params.id } });
+    res.status(200).json({'data':  blog});
+});
+
+// Update Blog
+routes.put('/update/:id', async (req, res) => {
+    try {
+        let blog = await blogModel.update(
+            { 
+                title: req.body.title,
+                description: req.body.description,
+                userId: req.body.userId,
+                status: req.body.status
+            },
+          { where: { id: req.params.id } }
+        );
+        res.status(200).json({'data': 'Record Updated Successfully.'});
+    } catch (err) {
+        res.json({'data': err});
+    }
+});
+
+// Soft Delete Blog
+routes.delete('/delete/:id', async (req, res) => {
+    try {
+        let blog = await blogModel.findOne({ where: { id: req.params.id } });
+        if(blog){
```

Comment:
please use `if (!blog) return res` where you need to verify object is not null.

Reply:
Used!